_api-reference/profile.md

```diff
@@ -0,0 +1,754 @@
+:--- | :--- 
+`create_weight` | A `Query` object in Lucene is immutable. Yet, Lucene should be able to reuse `Query` objects in multiple IndexSearchers. Thus, `Query` objects need to keep temporary state and statistics associated with the index in which the query is executed. To achieve reuse, every `Query` object generates a Weight object, which keeps the temporary context (state) associated with the <IndexSearcher, Query> tuple. The `create_weight` field contains the time spent in the process of creating the Weight object.
+`build_scorer` | A `Scorer` iterates over matching documents and generates a score for each document. The `build_scorer` field contains the time spent on generating the `Scorer` object. This does not include the time spent scoring the documents. The `Scorer` initialization time depends on the optimization and complexity of a particular query. The `build_scorer` parameter also includes the time associated with caching, if caching is applicable and enabled for the query.
+`next_doc` | The `next_doc` Lucene method returns a document ID of the next document that matches the query. This method is a special type of the `advance` method and is equivalent to `advance(docId() + 1)`. The `next_doc` method is more convenient for many Lucene queries. The `next_doc` field contains the time it takes to determine the next matching document, which varies depending on the query type.  
+`advance` | The `advance` method is a lower-level version of the `next_doc` method in Lucene. It also finds the next matching document but necessitates that the calling query perform additional tasks, such as identifying skips. Some queries, such as conjunctions (`must` clauses in Boolean queries), cannot use `next_doc`. For those queries, `advance` is timed.
+`match` | For some queries, matching documents is performed in two steps. First, the document is matched approximately. Second, those documents that are approximately matched are examined with a more comprehensive process. For example, a phrase query first checks if a document contains all terms in the phrase. Next, it verifies that the terms are in order (which is a more expensive process). The `match` field is non-zero only for those queries that use the two-step verification process. 
+`score` | Contains the time taken for a `Scorer` to score a particular document.
+`shallow_advance` | Contains the time for executing the `advanceShallow` Lucene method.
+`compute_max_score` | Contains the time for executing the `getMaxScore` Lucene method.
+`set_min_competitive_score` | Contains the time for executing the `setMinCompetitiveScore` Lucene method.
+`<method>_count` | Contains the number of invocations of a `<method>`. For example, `advance_count` contains the number of invocations of the `advance` method. Different invocations of the same method happen because the method is called on different documents. You can determine the selectivity of a query by comparing counts in different query components.
```

Comment:
Maybe we should change the "method" to "breakdownType"?

Reply:
If we change to breakdown type, do you have suggestions on how to change the description? Because you can't invoke a breakdown type, you can only invoke a method. All of these breakdown types correspond to similarly named methods, correct?

Reply:
Thanks for the clarification. Then keeping it as a "method" makes more sense to me.